### PR TITLE
Daylight: Date & Time inputs: edits

### PR DESCRIPTION
**Pull request by margaree**
These are Jeff's suggested changes for the date/time page. This opens the components on first view (not on subsequent due to changing properties in the properties table then causing jumpy behaviour). Related PR https://github.com/BrightspaceUI/documentation/pull/314 . 

Also I'm wondering if we should find a way to some hide scripts/styles we have from the demo code since some are more implementation details for display purposes and not necessarily for actual usage.

### CODE REVIEW

File: components/inputs/docs/input-date-time.md [hunk 1]
```diff
@@ -10,13 +12,37 @@ size:xlarge
...
 </script>
+<script>
+  window.addEventListener('load', function () {
+    const demoElem = document.querySelector('#demo-element');
+    if (!demoElem.hasAttribute('data-first-load')) return;
+
+    setTimeout(function() {
+      var input = document.querySelector('d2l-input-date');
+      input.opened = true;
+    }, 1000);
```
Comment (GeorgePlukov) [inline components/inputs/docs/input-date-time.md:23]: I think this should be 100

File: components/inputs/docs/input-date-time.md [hunk 2]
```diff
@@ -69,7 +108,7 @@ To make your usage of `d2l-input-date` accessible, use the following properties
 
 ## Date Range Input [d2l-input-date-range]
 
-The `<d2l-input-date-range>` component consists of two input-date components - one for the start of a range and one for the end of a range. Values specified for these components (through the `start-value` and/or `end-value` attributes) are displayed if specified, and selected values are reflected.
+Use the `<d2l-input-date-range>` component when users need to enter two dates in a range, like a course start and end date. The component consists of two input-date components - one for the start of a range and one for the end of a range. Values specified for these components (through the `start-value` and/or `end-value` attributes) are displayed if specified, and selected values are reflected.
```
Comment (GeorgePlukov) [inline components/inputs/docs/input-date-time.md:111]: I think the `Use <component> when ...` statements will be very useful for other devs!

File: components/inputs/docs/input-date-time.md [hunk 3]
```diff
@@ -289,10 +381,11 @@ To make your usage of `d2l-input-date-time` accessible, use the following proper
 | Attribute | Description |
 |--|--|
 | `label` | **REQUIRED** [Acts as a primary label on the input](https://www.w3.org/WAI/tutorials/forms/labels/) |
+| `label-hidden` | Use if label should be visually hidden but available for screen reader users |
 
 ## Date-Time Range Input [d2l-input-date-time-range]
 
-The `<d2l-input-date-time-range>` component consists of two input-date-time components - one for the start of a range and one for the end of a range. Values specified for these components (through the `start-value` and/or `end-value` attributes) are displayed if specified, and selected values are reflected.
+Use the `<d2l-input-date-time-range>` component when users need to enter a single date and time, like a due date. The component consists of two input-date-time components - one for the start of a range and one for the end of a range. Values specified for these components (through the `start-value` and/or `end-value` attributes) are displayed if specified, and selected values are reflected.
```
Comment (GeorgePlukov) [inline components/inputs/docs/input-date-time.md:388]: Is this the correct text to go under Date-time range?

**Review by GeorgePlukov — CHANGES_REQUESTED** [review]

File: components/inputs/docs/input-date-time.md (hunk 1 above)
Comment (margaree) [reply components/inputs/docs/input-date-time.md:23]: This one's purposefully 1000 in order to emphasize the opening

File: components/inputs/docs/input-date-time.md (hunk 3 above)
Comment (margaree) [reply components/inputs/docs/input-date-time.md:388]: I'll take a look!

**Review by GeorgePlukov — APPROVED** [review]

**Pull request merged**